recipe/meta.yaml

```
@@ -1,35 +1,47 @@
 {% set name = "slycot" %}
-{% set version = "0.3.3" %}
-{% set sha256 = "d98ec859a398af8a3917a0d62e0724996c8b3e780813f8685410525aed9fbafd" %}
+{% set version = "0.3.4" %}
+{% set sha256 = "3ae6db8df6b99b4417394404a548c6464248fc7031f21008f992e317f91d23f1" %}
```

```suggestion
{% set sha256 = "63e648b2be8c8ba0322a5fe286bdcf0e8d28a0c12e9ab1510f2a8725fc1283ec" %}
```

That's the pypi hash.